assignments/arrays.js

```diff
@@ -63,12 +63,15 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+console.log(`Car 33 is a ` + inventory[32].car_year +" " + inventory[32].car_make + " " +  inventory[32].car_model);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+lastCar [inventory.length-1];
```

Comment:
`lastCar` can be set to equal to the last car of the inventory like so
`lastCar = inventory[ inventory.length - 1 ];`
in this way you are reassigning `lastCar` to the last index of `inventory`.
`inventory[index]` any expression inside of an `[]` should evaluate to an index number
`inventory.length` gives us the total length of an array.
`inventory.length - 1`  gives us the last index of an array.